pkg/keymap/triggers/triggers.go

```diff
@@ -0,0 +1,39 @@
+package triggers
+
+import "github.com/kubeshop/testkube-operator/pkg/validation/tests/v1/testtrigger"
+
+type KeyMap struct {
+	Resources  []string            `json:"resources"`
+	Actions    []string            `json:"actions"`
+	Executions []string            `json:"executions"`
+	Events     map[string][]string `json:"events"`
+}
+
+func NewKeyMap() *KeyMap {
+	return &KeyMap{
+		Resources:  testtrigger.GetSupportedResources(),
+		Actions:    testtrigger.GetSupportedActions(),
+		Executions: testtrigger.GetSupportedExecutions(),
+		Events:     getSupportedEvents(),
+	}
+}
+
+func getSupportedEvents() map[string][]string {
```

Comment:
Should we hard code it or have something in helm, for example?

Reply:
I think it is fine to hardcode it as we need to add new code to support new events